grammar.js

```diff
@@ -2427,52 +2444,10 @@ module.exports = grammar({
       $._type,
     ),
 
-    interval_definitions: $ => repeat1(
```

Comment:
Why has this been removed? I makes the parser too liberal in my opinion.

Reply:
It make the parser more liberal and in the end I think you want the whole string highlighted as a string.

Reply:
it was only ever stringified in `interval` which does make it feel like a little bit of a reach imo, I think I'm fine with ditching it